### PR TITLE
add a docker-compose file

### PR DESCRIPTION
**Pull request by Fuseteam**
with docker-compose starting the container can be simplified to a simple `docker-compose up -d`
we could also add mount points for apk and the password files by default

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+version: '2'
```
Comment (AkihiroSuda) [inline docker-compose.yml]: Can we use version 3?

Comment (Fuseteam) [reply docker-compose.yml]: uh i can try, would it make a difference?

Comment (Fuseteam) [reply docker-compose.yml]: i see no regressions so i don't see why not

**Comment by AkihiroSuda** [comment]
Thanks

**Comment by Fuseteam** [comment]
yay

**Pull request merged**